lib/oli/authoring/media_library.ex

```diff
@@ -212,7 +212,9 @@ defmodule Oli.Authoring.MediaLibrary do
   end
 
   defp upload_file(bucket, file_name, contents) do
-    S3.put_object(bucket, file_name, contents, [{:acl, :public_read}]) |> ExAws.request()
+    mime_type = MIME.from_path(file_name)
```

Comment:
This defaults to application/octet-stream for unknown types, which is what we used to always use.